staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret_tls.go

```diff
@@ -179,10 +172,19 @@ func (o *CreateSecretTLSOptions) Complete(f cmdutil.Factory, cmd *cobra.Command,
 }
 
 // Validate checks if CreateSecretTLSOptions hass sufficient value to run
-func (o *CreateSecretTLSOptions) Validate() error {
+func (o *CreateSecretTLSOptions) Validate(cmd *cobra.Command) error {
 	// TODO: This is not strictly necessary. We can generate a self signed cert
 	// if no key/cert is given. The only requirement is that we either get both
 	// or none. See test/e2e/ingress_utils for self signed cert generation.
+	requiredFlags := []string{"cert", "key"}
+	for _, requiredFlag := range requiredFlags {
+		if value := cmdutil.GetFlagString(cmd, requiredFlag); len(value) == 0 {
+			return cmdutil.UsageErrorf(cmd, "flag %s is required", requiredFlag)
+		}
+	}
```

Comment:
You can remove this, you already validate below :)

Reply:
Oops